appcues/src/main/java/com/appcues/AppcuesCustomComponentView.kt

```diff
@@ -0,0 +1,29 @@
+package com.appcues
+
+import android.view.ViewGroup
+
+/**
+ * AppcuesCustomComponentView is an interface to be implemented when registering a new custom component linked to an identifier.
+ *
+ * (static call) Appcues.registerCustomComponent("identifier", appcuesCustomComponentViewImpl)
+ */
+public interface AppcuesCustomComponentView {
+
+    /**
+     * Define your properties here as a map with example values so it can be
+     * previewed in debugger
+     */
+    public val debugConfig: Map<String, Any>
```

Comment:
should this be optional? I think on the iOS example code I saw it was planned optional.

Reply:
actually yes